### PR TITLE
update release artifacts

### PR DESCRIPTION
**Pull request by smacker**
Fix: #94 

- Fixed gemini docker file
- Copy python source code in tarball
- Bash script to run features-extractor (don't reuse it in docker file. Both are very-very simple and there are like 2 similar commands so unification would require much more code than just keep it as is) 
- Build image & upload image for features-extractor
- Check for python in `report`
- Update sourced-ml to 0.4.5 (0.4.4 contains a strange version of spark and it gets re-downloaded for every run)
- Add  [coursier](https://github.com/coursier/coursier) for faster dependency download

We will need a bit more work after `report` is ready (most probably install python in gemini image, its dependencies and maybe update some scripts). I left it out of this PR for now.

### CODE REVIEW

File: feature_extractor [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Convenience script to run features extraction service locally
+
+service_dir="src/main/python/feature-extractor"
+
+hash python3 >/dev/null 2>&1 || { echo "Please install python3" >&2; exit 1; }
+
+if [ -z "$VIRTUAL_ENV" ]; then
+    echo "Looks like you don't use virtual environment for python"
+    echo "Please active it or create a new one"
+    echo "Move information is here: https://virtualenv.pypa.io/en/stable/"
```
Comment (carlosms) [inline feature_extractor]: Small typos:
"Please **activate** it or create a new one"
"**More** information here: https://virtualenv.pypa.io/en/stable/"

**Comment by smacker** [comment]
it's weird that pip couldn't resolve grpcio correctly, constraints are correct...
Anyway, I just set the same version as in bblfsh and now it's ok.

Typos have been fixed.

Further updates will be done after #103 is merged.

**Comment by smacker** [comment]
Python & py deps for report added to Dockerfile. Should be good to go now.

**Comment by bzz** [comment]
```
 Ran for 12 min 5 sec
 Total time 37 min 3 sec
```

🤔

**Comment by bzz** [comment]
Sorry, I do not think this change have to do with test times.

Looks great to me! Let's merge

**Pull request merged**